index.d.ts

```diff
@@ -121,7 +121,7 @@ const array3 = new Uint8Array([7, 8, 9]);
 export function compareUint8Arrays(a: Uint8Array, b: Uint8Array): 0 | 1 | -1;
 
 /**
-Convert a `Uint8Array` (containing a UTF-8 string) to a string.
+Convert a `Uint8Array` to a string using an optional encoding (defaults to UTF-8).
```

Comment:
This does not make it clear enough that the encoding parameters is about the assumed source encoding of the bytes for the string. It also needs to document which other encodings it supports or at least link the MDN docs about it.